src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -1547,15 +1548,30 @@ private static GeneratableElement getExampleFor(ModElement modElement, boolean u
 					VariableTypeLoader.BuiltInTypes.NUMBER.getName() :
 					VariableTypeLoader.BuiltInTypes.LOGIC.getName());
 			return gamerule;
+		} else if (ModElementType.VILLAGERPROFESSION.equals(modElement.getType())) {
+			VillagerProfession profession = new VillagerProfession(modElement);
+			profession.displayName = modElement.getName();
+			List<MItemBlock> poiBlocks = ElementUtil.loadAllPOIBlocks(modElement.getWorkspace(), null);
+			profession.pointOfInterest = new MItemBlock(modElement.getWorkspace(), getRandomMCItem(random,

```

Comment:
This and `actionSound` should check for `emptyLists` I think.

Reply:
Not sure what this means

Reply:
OKay, nevermind, these fields are required and other required fields do ignore `emptyLists`.